internal/timeutil/time_test.go

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (C) 2019 Vdaas.org Vald team ( kpango, kmrmt, rinx )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package timeutil
+
+import (
+	"testing"
+	"time"
+)
+
+func TestParse(t *testing.T) {
+	type args struct {
+		t string
+	}
+	tests := []struct {
+		name    string
+		args    args
```

Comment:
in this case we don't need args struct, please use premetive type instead of struct

Reply:
fixed